docs/reference/ml/df-analytics/apis/get-dfanalytics.asciidoc

```diff
@@ -33,38 +33,42 @@ information, see {stack-ov}/security-privileges.html[Security privileges] and
 ==== {api-description-title}
 
 You can get information for multiple {dfanalytics-jobs} in a single API request 
-by using a comma-separated list of {dfanalytics-jobs} or a wildcard expression. 
-You can get information for all {dfanalytics-jobs} by using _all, by specifying 
-`*` as the `<data_frame_analytics_id>`, or by omitting the
-`<data_frame_analytics_id>`.
+by using a comma-separated list of {dfanalytics-jobs} or a wildcard expression.
 
 [[ml-get-dfanalytics-path-params]]
 ==== {api-path-parms-title}
 
 `<data_frame_analytics_id>`::
   (Optional, string) Identifier for the {dfanalytics-job}. If you do not specify
   one of these options, the API returns information for the first hundred
-  {dfanalytics-jobs}.
-  
-`allow_no_match` (Optional)::
-  (boolean) If `false` and the `data_frame_analytics_id` does not match any 
-  {dfanalytics-job} an error will be returned. The default value is `true`.
+  {dfanalytics-jobs}. You can get information for all {dfanalytics-jobs} by 
+  using _all, by specifying `*` as the `<data_frame_analytics_id>`, or by 
+  omitting the `<data_frame_analytics_id>`.
 
 [[ml-get-dfanalytics-query-params]]
 ==== {api-query-parms-title}
 
 `allow_no_match`::
-  (Optional, boolean) If `false` and the `data_frame_analytics_id` does not
-  match any {dfanalytics-job} an error will be returned. The default value is
-  `true`.
+  (Optional, boolean) Specifies what to do when the request:
++
+--
+* Contains wildcard expressions and there are no {dfanalytics-jobs} that match.
+* Contains the `_all` string or no identifiers and there are no matches.
+* Contains wildcard expressions and there are only partial matches. 
+
+The default value is `true`, which returns an empty `data_frame_analytics` array 
+when there are no matches and the subset of results when there are partial 
+matches. If this parameter is `false`, the request returns a `404` status code 
+when there are no matches or only partial matches.
+--
 
 `from`::
   (Optional, integer) Skips the specified number of {dfanalytics-jobs}. The
-  default value is `0`.
+  default value is `0`. It could be also used as a path parameter.
```

Comment:
FYI we added comments in https://raw.githubusercontent.com/elastic/docs/master/shared/api-ref-ex.asciidoc to clarify the different parameter sections. Path parameters are used before the ? in the request, so I don't think this is a valid change. I think you *can* put some of these in the request body too, but I don't think it's worth repeating them there. 
```suggestion
  default value is `0`.
```